src/main/daml/Daml/Finance/Util/Date/DayCount.daml

```diff
@@ -223,3 +253,17 @@ isEndOfFebruary (y, m, d) =
 getDateEndOfFebruary : Int -> Date
 getDateEndOfFebruary y =
   fromGregorian (y, Feb, monthDayCount y Feb)
+
+-- | HIDE
+leapDayInDateRange : Int -> Date -> Date -> Bool
+leapDayInDateRange y fromDate toDate =
+  endOfFebDate >= fromDate && endOfFebDate <= toDate && isLeapYear y
+  where endOfFebDate = getDateEndOfFebruary y
+
+-- | HIDE
+countLeapDays : Date -> Date -> Int
+countLeapDays fromDate toDate =
+  foldl (\acc y -> acc + if leapDayInDateRange y fromDate toDate then 1 else 0) 0 [y1..y2]
```

Comment:
Just fyi, changing `leapDayInDateRange` to return the `Int` instead of the `Bool` would allow for writing

```suggestion
  foldMap (\y -> leapDayInDateRange y fromDate toDate) [y1..y2]
```

Reply:
Looks nice, but it gave me an error `No instance for (Monoid Int) arising from a use of ‘foldMap’`.

Reply:
aaah, I thought `Int` implemented `Monoid` with `0` as the base case. It doesn't look to be the case though ... https://stackoverflow.com/questions/29499119/why-int-does-not-implement-monoid